webapp/types/tunnel.ts

```diff
@@ -93,6 +93,8 @@ export const enum EvmDepositStatus {
   DEPOSIT_TX_FAILED = 4,
   // Approval failed
   APPROVAL_TX_FAILED = 5,
+  // Relayed
```

Comment:
I'll add this for clarity
```suggestion
  // Funds have been minted in Hemi
```